src/sum.js

```diff
@@ -0,0 +1,11 @@
+// create variable <a> with value <11>
+// create variable <b> with value <7>
+// create variable <sum> and store the sum of <a> and <b>
+// log in terminal string "11 + 7 = 18" using all variables and template strings
+
+const a = 11;
+const b = 7;
+const sum = a + b;
+
+console.log(a + " + " + b + " = " + (a + b));
```

Comment:
@RomanPantya 
there is any task about first console.log(...)
only second variant is required
delete line number 10